src/foam/core/lib.js

```diff
@@ -23,11 +23,14 @@ foam = {
   isServer: globalThis.FOAM_FLAGS.node,
   core:     {},
   util:     {
-    path: function(root, path, opt_ensure) {
+    path: function(root, path, opt_ensure, strict) {
```

Comment:
Isn't this the reverse of opt_ensure and we should just return undefined if we walk off the end?

Reply:
Root is never going to undefined though because we always assign it an empty object